docs/release-notes/february-2024.md

```diff
@@ -0,0 +1,172 @@
+* Added the ability to distinguish between which rules are available to Semgrep
+  Pro Engine users and which ones are available to Semgrep OSS users.
+* Added support for nested record patterns on the left-hand side of an
+  assignment.
+* `metavariable-regex` can now match on metavariables of interpolated strings
+  that use variables with known values.
+* Added support for parsing Swift Package Manager manifest files and lock files.
+* **Taint analysis**:
+  * Added support for Python constructors
+  * Added support for index sensitivity. Semgrep tracks taint on individual
+    indexes of a data structure when these are constant values, either integers
+    or strings, and the code uses the built-in syntax for array indexing.
+  * Added `exact: false` so you can specify that anything inside a code
+    region is a sink.
+  * When `exact: true` and `taint_assume_safe_functions: true`, Semgrep now
+    considers that, if the specified formula isn't a `patterns` with a
+    `focus-metavariable`, it must look for taint in the function call's arguments.
+
+### Changed
+
+* Improved error handling during inter-file analysis so Semgrep Code doesn't crash.
+* **CLI**: If there are multiple errors resulting from the user running Pro
+  rules without a paid subscription, the CLI groups all errors and reports a
+  single warning.
+* The project name for repositories scanned locally is `local_scan/<repo_name>`
+  instead of `<repo_name>`.
+* The **View Results** URL displayed for findings now includes the repository
+  and branch names.
+
+### Fixed
+
+* Fixed issues with trailing newline parsing in `pyproject.toml` and
+  `poetry.lock` files.
+* Fixed an issue with incorrect autofix application where multiple fixes were
+  applied to the same line.
+* Fixed issue where tokens for type parameter brackets weren't stored correctly.
+  They're now stored in the generic AST, allowing Semgrep to autofix
+  these constructs correctly.
+* Fixed an issue where Semgrep doesn't support multiple labels for taint
+  traces. Now, Semgrep looks at the `requires` of the sink, and if it has the
+  shape `A and ...`, it picks `A` as the preferred label and reports the
+  trace.
+* Fixed issue where taint signatures don't capture changes to parameter fields.
+* Scan summary links printed after users run `semgrep ci` now reflect a
+  custom `SEMGREP_APP_URL` if set.
+
+## ⛓️ Supply Chain
+
+### Added
+
+* Added the ability to filter for dependencies that Semgrep has commented on.
+  <!-- https://github.com/semgrep/semgrep-app/pull/12898 -->
+* Added manual review advice to GitHub PR comments. Certain Semgrep Supply Chain (SSC) findings require **manual review** to verify if the finding is reachable or not. GitHub PR comments now include this advice to help you ascertain if the finding is reachable or not. <!-- 12907 -->
+* Separated `reachable_if` and `upgrade_only` exposures. Users can better determine a finding's criticality through this distinction. <!-- 12657 -->
```

Comment:
Sara is waiting for clarification if the separation appears to the end user in some way in the UI.

Reply:
It is not visible in the UI. removing.